Feisty/FriendsView/FriendsViewController.swift

```diff
@@ -62,10 +62,14 @@ class FriendsViewController: UITableViewController {
   
   func createAndShowAlert(message: String, handler: @escaping((UIAlertAction) -> Void)) {
     
-    let alertController = UIAlertController(title: "Error", message: "", preferredStyle: .alert)
-    alertController.message = message
-    alertController.addAction(UIAlertAction(title: "Dismiss", style: .default, handler: handler))
-    self.present(alertController, animated: true, completion: nil)
+    DispatchQueue.main.async {
+      
+      let alertController = UIAlertController(title: "Error", message: "", preferredStyle: .alert)
```

Comment:
Just a suggestion, if you plan on having error alerts in the future, would be a good idea to make a generic view extension that takes an error message and displays it. You can customize with taking closures as well if you want to do extensions for other alerts. Not necessary to implement if you're not using it in a lot of places though

Reply:
That is a fantastic idea for the error propagation that is planned. Was thinking how I would do it and that is great.